java/memory/memory-core/src/main/java/org/apache/arrow/memory/util/MemoryUtil.java

```diff
@@ -141,7 +141,7 @@ public Object run() {
       // This exception will get swallowed, but it's necessary for the static analysis that ensures
       // the static fields above get initialized
       final RuntimeException failure = new RuntimeException(
-          "Failed to initialize MemoryUtil. Was Java started with " +
+          "Failed to initialize MemoryUtil. Was Java started without " +
```

Comment:
Maybe just make it imperative to be absolutely clear? "You must start Java with ..."